assets/src/web-stories-block/block/components/storiesBlockConfigurationPanel.js

```diff
@@ -99,38 +39,53 @@ function BlockConfigurationPanel({
     <Placeholder
       icon={<BlockIcon icon={icon} showColors />}
       label={label}
-      instructions={instruction}
-      data-testid="ws-block-configuration-panel"
+      instructions={instructions}
+      className="web-stories-block-configuration-panel"
     >
-      <TypeGrid $columnCount={columnCount}>
+      {/*
+       * Disable reason: The `list` ARIA role is redundant but
+       * Safari+VoiceOver won't announce the list otherwise.
+       */
+      /* eslint-disable-next-line jsx-a11y/no-redundant-roles */}
+      <ul
+        className="web-stories-block-configuration-panel__options"
+        role="list"
+        aria-label={__('Block Types', 'web-stories')}
+      >
         {selectionOptions.map((option) => (
-          <TypeCard
-            className="components-card"
-            key={option.id}
-            onClick={() => {
-              setAttributes({ [selectionType]: option.id });
-            }}
-          >
-            <TypeMedia className="components-card__media">
-              {'viewType' === selectionType && <Icon icon={option.panelIcon} />}
-              {'blockType' === selectionType && <Icon icon={option.icon} />}
-            </TypeMedia>
-            <TypeCardBody className="components-card__body">
+          <li key={option.id}>
+            <Button
+              variant="secondary"
+              isSecondary
+              onClick={() => {
+                setAttributes({ [selectionType]: option.id });
+              }}
+            >
+              {'viewType' === selectionType && (
+                <Icon icon={option.panelIcon} title={option.label} />
+              )}
+              {'blockType' === selectionType && (
+                <Icon icon={option.icon} title={option.label} />
+              )}
+            </Button>
+            <span
+              className="web-stories-block-configuration-panel__label components-placeholder__instructions"
+              role="presentation"
+            >
               {option.label}
-            </TypeCardBody>
-          </TypeCard>
+            </span>
+          </li>
         ))}
-      </TypeGrid>
+      </ul>
```

Comment:
Comparing to https://github.com/WordPress/gutenberg/blob/63857ac31fbcbeb1001f8f23ab1281e5ff685710/packages/block-editor/src/components/block-variation-picker/index.js#L36-L60

Can't we pass an icon to the button? A label and icon size would likely not go a miss here.

Reply:
I'll check